src/core/resources/transactions/consolidatedTransactions.ts

```diff
@@ -145,11 +145,6 @@ async function parseConsolidatedTransactions(
       }),
     )
     .filter(Boolean);
-
-  const parsedConsolidatedTransactions = (
-    await Promise.all(parsedTransactionPromises)
```

Comment:
parseTransaction is not a promise

Reply:
we have the same here too https://github.com/rainbow-me/browser-extension/blob/master/src/core/resources/transactions/transactions.ts#L119

Reply:
would be nice to remove as well